netcdf_to_cb/run_ingest_threads.py

```diff
@@ -166,9 +166,12 @@ def runit(self, args):  # pylint:disable=too-many-locals
             self.file_pattern = args["file_pattern"].strip()
         try:
             # put the real credentials into the load_spec
+            logging.info("getting cb_credentials")
             self.cb_credentials = self.get_credentials(self.load_spec)
             # establish connections to cb, collection
+            logging.info("cb_credentials are: ${self.cb_credentials}")
```

Comment:
We should probably remove this so we don't log our secrets in plaintext.

Reply:
Thanks for catching that. Done.